docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  #  Create a service named db.
+  db:
+    image: "postgres:12-alpine"
+    container_name: "pipeline_db"
+    #   Setup the username, password, and database name
+    environment:
+      - POSTGRES_USER=pipeline
+      - POSTGRES_PASSWORD=pipeline
+      - POSTGRES_DB=pipeline
+    #   Map port 6432 (localhost) to port 5432 on the container
+    ports:
+      - "5432:5432"
+
+  gitlab:
+    image: 'gitlab/gitlab-ce:latest'
+    restart: always
+    environment:
+      GITLAB_OMNIBUS_CONFIG: |
+        external_url 'http://localhost:9080'
+        nginx['listen_port'] = 9080                   # make nginx to listen on the same port as confgured in external_url
+        # Add any other gitlab.rb configuration here, each on its own line
+    ports:
+      - '9080:9080'
+      - '9443:443'
+      - '9022:22'
+    volumes:
+      - '/srv/gitlab/config:/etc/gitlab'
+      - '/srv/gitlab/logs:/var/log/gitlab'
+      - '/srv/gitlab/data:/var/opt/gitlab'
+
+  cromwell_pipeline:
+    container_name: cromwell_pipeline
+    build:
+      context: cmwl_pipeline
+      dockerfile: Dockerfile
+    command: java -jar portal.jar
+    environment:
+      - CMWL_DB_SERVER_NAME=pipeline_db
+      - CMWL_DB_PORT_NUMBER=5432
+      - CMWL_DB_USER=pipeline
+      - CMWL_DB_PASSWORD=pipeline
+      - CMWL_DB_NAME=pipeline
+    ports:
+      - 8080:8080
+    depends_on:
+      - db
```

Comment:
I suggest we add `gitlab` as a dependency as well since our application can't work without it.